### PR TITLE
Submitting my a11y-fails page with fixes

### PR DESCRIPTION
**Pull request by tannerdolby**
Hi @5t3ph - This was a really fun project! I will definitely participate if you create another one in the future.

Also, including the [suggested fixes commit](https://github.com/5t3ph/a11y-fails/pull/1/commits/3480f611c7f2adc03effb7da9f32d57ed7adc7e5) was super helpful. I followed the WCAG and other links you included and learned a lot while fixing things.

I apologize for so many commits in one PR, I tried to keep each commit clear and cohesive.

## Proposed Fixes:
* users with visual impairments - ✅
* users of assistive technology - ✅
* users with motion sensitivities - ✅
* users who perform keyboard interaction - ✅

## Tools Used
* https://wave.webaim.org/
* https://webaim.org/resources/contrastchecker/

### Notes
I didn't deploy my fork on Netlify so I wasn't actually able to run the [WAVE Report](https://wave.webaim.org/) after my a11y fixes to catch any potential fixes still present. Please let me know if there is anything I missed or could rework. Thank you Stephanie!

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -3,82 +3,90 @@
...
   </head>
   <body>
-    <ul class="nav">
-      <li><a href="/about">About</a></li>
-      <li><a href="/contact">Contact</a></li>
-      <li><a href="/products">Products</a></li>
-    </ul>
-    <div class="header">
+    <a href="#main" class="skip-link">Skip to main content</a>
+    <nav role="navigation">
```
Comment (5t3ph) [inline src/index.html]: The `role` is redundant in this case 👍 Use of `nav` is sufficient to indicate the role.

File: src/index.html [hunk 2]
```diff
@@ -3,82 +3,90 @@
...
+    </nav>
+    <header class="header">
       <h1><span>a11y</span> Fails</h1>
-    </div>
-    <div class="main">
-      <h3>Product List</h3>
-      <aside class="products">
-        <div class="card">
+    </header>
+    <main class="main" role="main" id="main" tabindex="-1">
```
Comment (5t3ph) [inline src/index.html]: Same idea here as `nav` where `role` is redundant 👍

File: src/index.html [hunk 3]
```diff
@@ -3,82 +3,90 @@
...
           cols="30"
           rows="3"
-          placeholder="Message"
         ></textarea>
 
-        <button class="button" role="button">Submit</button>
-      </div>
-    </div>
-    <div class="footer">
+        <button class="button" role="submit">Submit</button>
```
Comment (5t3ph) [inline src/index.html]: Small error here!

```suggestion
        <button class="button" type="submit">Submit</button>
```

File: src/index.html [hunk 4]
```diff
@@ -3,82 +3,90 @@
...
           </p>
-          <button class="button">Buy Now</button>
-        </div>
-      </aside>
+          <a href="#" class="button" aria-label="Buy Item 3 Now">Buy Now</a>
+        </li>
+      </ul>
 
-      <div class="form">
+      <form class="form">
```
Comment (5t3ph) [inline src/index.html:68]: Probably the only additional thing I would suggest is adjusting the spacing between form fields to provide a stronger visual indication of which label belongs to which field 😊

**Review by 5t3ph — CHANGES_REQUESTED** [review]
Great job on this! A couple minor things noted for your future reference! I'm glad it was a good practice experience for you 😊

**Comment by 5t3ph** [comment]
Oh and your preview is viewable: https://deploy-preview-6--a11y-fails.netlify.app/

File: src/index.html (hunk 3 above)
Comment (tannerdolby) [reply src/index.html]: Good catch! I saw this in your suggested changes and it makes sense. Thank you! Fixed with commit [b6d61a5](https://github.com/5t3ph/a11y-fails/pull/6/commits/b6d61a5b334937b40cd9714c8ebb1ae418466ad5)

File: src/index.html (hunk 2 above)
Comment (tannerdolby) [reply src/index.html]: Thanks 👍 fixed with commit [82fa62b](https://github.com/5t3ph/a11y-fails/pull/6/commits/82fa62b4223eca14eafe8f8487b8d3439126fd5c)

File: src/index.html (hunk 1 above)
Comment (tannerdolby) [reply src/index.html]: Ok thank you! I've seen this redundant usage around the web so I'm glad to understand now that landmark elements like `<nav>` natively have the role assigned. I made sure to remove this with commit [129dbf1](https://github.com/5t3ph/a11y-fails/pull/6/commits/129dbf119642d5fb2ce1070d9c0710ea97d58f27)

File: src/index.html (hunk 4 above)
Comment (tannerdolby) [reply src/index.html:68]: I agree! I added 16px of bottom spacing to each form field and I think it helps to indicate each corresponding label/field more clearly. Fixed with commit [a2ec76e](https://github.com/5t3ph/a11y-fails/pull/6/commits/a2ec76e0c4bfcff4ce6e14b1a693d1184436e897)

**Comment by tannerdolby** [comment]
@5t3ph I've fixed the remaining issues you mentioned with my most recent 4 commits. Let me know if there is anything else I can update 😃

Thanks again!

**Comment by tannerdolby** [comment]
Also, I meant to ask you this in my previous comment but forgot. Will this PR end up being merged with [`steph:main`](https://github.com/5t3ph/a11y-fails) or am I able to close it now?

**Pull request closed without merging**